src/main/java/com/puppycrawl/tools/checkstyle/api/AutomaticBean.java

```diff
@@ -362,7 +362,7 @@ private static class RelaxedStringArrayConverter implements Converter {
         public Object convert(Class type, Object value) {
             // Convert to a String and trim it for the tokenizer.
```

Comment:
can you remove this comment too please? it states the obvious

Reply:
Done.